apps/antalmanac/src/components/RightPane/AddedCourses/NotificationsTabs.tsx

```diff
@@ -0,0 +1,60 @@
+import { TabContext, TabList, TabPanel } from '@mui/lab';
+import { Box, Tab, Paper } from '@mui/material';
+import { useMemo, useState } from 'react';
+import { useShallow } from 'zustand/react/shallow';
+
+import { NotificationsTable } from '$components/RightPane/AddedCourses/NotificationsTable';
+import { useNotificationStore } from '$stores/NotificationStore';
+
+function groupNotificationsByTerm(notifications: Record<string, unknown>) {
+    return Object.keys(notifications).reduce<Record<string, string[]>>((groups, key) => {
+        const parts = key.split(' ');
+        const term = parts.slice(-2).join(' ');
+        if (!groups[term]) {
+            groups[term] = [];
+        }
+        groups[term].push(key);
+        return groups;
+    }, {});
+}
+
+export function NotificationsTabs() {
+    const initialized = useNotificationStore(useShallow((store) => store.initialized));
+    const notifications = useNotificationStore(useShallow((store) => store.notifications));
+
+    const groups = useMemo(() => groupNotificationsByTerm(notifications), [notifications]);
+    const sortedTerms = useMemo(() => Object.keys(groups).sort(), [groups]);
+
+    const [activeTab, setActiveTab] = useState(sortedTerms.at(0));
+    const handleTabChange = (_: React.SyntheticEvent, newValue: string) => {
+        setActiveTab(newValue);
+    };
+
+    if (!activeTab) {
+        return null;
+    }
+
+    if (!initialized) {
+        return 'Loading notifications...';
+    }
+
+    return (
+        <Box sx={{ width: '100%' }}>
+            <TabContext value={activeTab}>
+                <Paper elevation={0} variant="outlined" square>
+                    <TabList onChange={handleTabChange} indicatorColor="primary" variant="fullWidth" centered>
```

Comment:
Is there a reason why the notifications need to be paginated instead of scrolling?